yaml/annotations.adoc

```diff
@@ -136,3 +136,39 @@ Applies to:
 // Applies to:
 // * StatefulSets
 
+
+// Type: procedure
+[id="service-yaml"] 
+= Creating a service when service-sync is disabled
+
+If you set `service-sync: "false"` when creating a site and you want to consume a service exposed on another site:
+
+. Create `backend.yaml` similar to the following:
++
+[source, yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: backend
+  annotations:
+    skupper.io/proxy: tcp
+    skupper.io/address: backend
```

Comment:
It may be interesting to mention that this needs to match the name of the service created on the other side, as that will be the main alteration the user will do on this file.